arkane/common.py

```diff
@@ -274,15 +275,23 @@ def load_yaml(self, path, species, pdep=False):
         freq_data = None
         if 'imaginary_frequency' in data:
             freq_data = data['imaginary_frequency']
-            del data['imaginary_frequency']
+            data = {key: data[key] for key in data.iterkeys() if key != 'imaginary_frequency'}
+        if not data['is_ts']:
+            if 'smiles' in data:
+                data['species'] = Species().fromSMILES(data['smiles'])
+            elif 'adjacency_list' in data:
+                data['species'] = Species().fromAdjacencyList(data['adjacency_list'])
```

Comment:
would you like to add inchi as well? Maybe using something like:
`Species(molecule=[Molecule().fromInChI().generate_resonance_structures()])`
or is there a simpler method?

Reply:
You can now do `Species(InChI='whatever')` and `Species(SMILES='whatever')` for instantiation.

Reply:
Thanks @mliu49 ! I'll implement this then